UnitTests/GitCommands.Tests/Git/GetAllChangedFilesOutputParserTest.cs

```diff
@@ -35,5 +37,14 @@ public async Task TestGetStatusChangedFilesFromString(string testName, string st
             await Verifier.VerifyJson(JsonConvert.SerializeObject(statuses))
                 .UseParameters(testName);
         }
+
+        [Test]
+        public async Task GitStatusDefault()
+        {
+            GitItemStatus item = GitItemStatus.Default("filename.txt");
+
+            await Verifier.VerifyJson(JsonConvert.SerializeObject(item))
+                .UseParameters("GitStatusDefault");
```

Comment:
🤔  You shouldn't need this line. It's needed if you have multiple input args, and you wish to use only a subset of those.

Reply:
removed